src/UIKit/UIEnums.cs

```diff
@@ -2866,4 +2880,44 @@ public enum UIKeyboardHidUsage : long {
 		KeyboardHiragana = KeyboardLang4,
 		KeyboardZenkakuHankakuKanji = KeyboardLang5,
 	}
+
+	[Flags, NoWatch, NoTV, iOS (13,4)]
+	[Native]
+	public enum UIEventButtonMask : long {
+		Primary = 1L << 0,
+		Secondary = 1L << 1,
+	}
+
+	[Flags, TV (13,4), NoWatch, iOS (13,4)]
+	[Native]
+	public enum UIAxis : ulong {
+		Neither = 0uL,
+		Horizontal = 1uL << 0,
+		Vertical = 1uL << 1,
+		Both = (Horizontal | Vertical),
+	}
+
+	[NoWatch, NoTV, iOS (13,4)]
+	[Native]
+	public enum UIScrollType : ulong {
```

Comment:
normally enums are `NSInteger`, so `long` (not unsigned)

Reply:
done